### DISCUSSION TITLE
chat shout

### DISCUSSION THREAD

**Discussion by RedAlphard** [Zrips/CMI#2365] [Q&A]
**Question: **
Hi Zrips,
I wanted to ask you if you can improve the usefulness of the chat shout; I would like that if a staffer activates the socialspy he can read both the chat shout and the local chat with a specific tag settable from the Locale_EN.yml file. The chat.range.bypass permission forces the staff to always write globally even when they would like to write only to a player within a 1000 block range; essentials provides this you can do it? otherwise I'll have to use a separate plugin for chat management and I don't like it.

Yours sincerely;
RedAlphard

**Comment by CloudeLecaw** [comment]
you can use chatshout if your useing chtchannels
you allready can setup the shouting range in config

would be nice if you close the ticket ;)
